e2e/pkg/dump/dump.go

```diff
@@ -46,6 +47,41 @@ func dumpResourceInAllNamespaces(ctx context.Context, resource client.ObjectList
 		return err
 	}
 
+	return dumpCleanUnstructuredOrResource(resource)
+}
+
+func dumpCleanUnstructuredOrResource(resource client.ObjectList) error {
+	if err := dumpCleanUnstructured(resource); err == nil {
+		return nil
+	}
+	return dumpResource(resource)
+}
+
+func dumpCleanUnstructured(resource client.ObjectList) error {
+	ur, err := runtime.DefaultUnstructuredConverter.ToUnstructured(resource)
+	if err != nil {
+		return err
+	}
+
+	ii := ur["items"].([]interface{})
+	for _, i := range ii {
+		ie, ok := i.(map[string]interface{})
+		if !ok {
+			continue
+		}
+
+		om, ok := ie["objectmeta"].(map[string]interface{})
+		if !ok {
+			continue
+		}
+
+		om["managedfields"] = nil
+	}
+
+	return dumpResource(ur)
+}
```

Comment:
It might be more efficient to change these in the structured version.  Why go through the trouble of converting that field to unstructured only to remove it?  I know `managedFields` can get fairly large.